src/components/forms/EditServiceForm.js

```diff
@@ -0,0 +1,270 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import { Form, Input, Select, InputNumber, DatePicker, Modal } from 'antd';
+import { STATUSES } from '../../const';
+import { UNIT_OPTIONS } from '../../const';
+import {
+  getServiceProviders,
+  editServiceAction,
+} from '../../state/actions/serviceActions';
+
+const { TextArea } = Input;
+
+function EditServiceForm({
+  visible,
+  onEdit,
+  onCancel,
+  serviceProviders,
+  recipients,
+  serviceTypes,
+  service,
+}) {
+  const [form] = Form.useForm();
+  form.resetFields();
+
+  return (
+    <>
+      <Modal
+        visible={visible}
+        title="Edit Service **NOT YET FUNCTIONAL**"
+        okText="Update Service"
+        cancelText="Cancel"
+        width="80%"
+        onCancel={onCancel}
+        onOk={() => {
+          form
+            .validateFields()
+            .then(values => {
+              form.resetFields();
+              onEdit(service.service_entry_id, values);
+            })
+            .catch(info => {
+              console.log('Validate Failed', info);
+            });
+        }}
+      >
```

Comment:
This is a lot of code without a commit. I would like to see more commits while you are working!